woocommerce/woocommerce-android
# "Add Variation" Button

Summary
==========
Address issue #3205 with two changes:

1 - Adapting the Variations `ProductDetailCardBuilder` method to now present the `Add Variation` text instead of the previous `This product has no variation yet` description.

2 - Adding to `VariationListFragment`a new kind of `ActionableEmptyView` that presents the `Add Variation` view and action instead of suggesting the user go to the web.

⚠️ The `Add Variation` button behavior will be implemented on upcoming PRs

How to Test
==========

New behavior (only available when on debug mode)
----------
| Click on Products section and select a Variable Product | Click on `Add Variations` and make sure no view is called instead of the old web redirection suggestion |
| ------------- | ------------- |
| ![Screenshot_1614251390](https://user-images.githubusercontent.com/5920403/109145190-ecd61f00-7740-11eb-8eb5-0a7a0158924a.png) | ![Screenshot_1614247056](https://user-images.githubusercontent.com/5920403/109145672-8e5d7080-7741-11eb-9b1a-f987efb38b82.png) |
Old behavior
----------
| Click on Products section and select a Variable Product | Click on `Variations` | Make sure presented View informs the user about adding variations only available on the web |
| ------------- | ------------- | ------------- |
| ![Screenshot_1614251390](https://user-images.githubusercontent.com/5920403/109145190-ecd61f00-7740-11eb-8eb5-0a7a0158924a.png) | ![Screenshot_1614251359](https://user-images.githubusercontent.com/5920403/109145594-7128a200-7741-11eb-925c-3d20a05a3cdf.png) | ![Screenshot_1614251362](https://user-images.githubusercontent.com/5920403/109145623-7a197380-7741-11eb-8081-fb7562e1837d.png) |

Update release notes:
